app/src/main/java/com/mad41/tripreminder/Firebase/WriteHandler.java

```diff
@@ -0,0 +1,43 @@
+package com.mad41.tripreminder.Firebase;
+
+import androidx.lifecycle.LiveData;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.mad41.tripreminder.room_database.MyRoomDataBase;
+import com.mad41.tripreminder.room_database.trip.Trip;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class WriteHandler{
+
+
+    public static void  WriteInfireBase(List<Trip> trips) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.